export-samples/html/cv-galactic-tech-innovations.html

```diff
@@ -0,0 +1,692 @@
+        row-gap: var(--spacing_32);
+        column-gap: var(--spacing_32);
+      }
+      @media (min-width: 728px) {
+        body {
+          padding: var(--spacing_64);
+        }
+        .container {
+          column-gap: var(--spacing_32);
+        }
+      }
+      @media (min-width: 1024px) {
+        .container {
+          grid-template-areas:
+            'header header'
+            'aside main'
+            'aside main';
+          max-width: 1220px;
+          align-self: center;
+        }
+      }
+
+      /* header styles */
+      header {
+        grid-area: header;
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        gap: var(--spacing_4);
+        align-self: stretch;
+      }
+      .header-title {
+        min-width: 100%;
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        flex: auto;
+        /*justify-content: space-between;*/
+      }
+      .header-name {
+        display: inline-block;
+        /*min-width: fit-content;*/
+        font-family: Mohave;
+        font-size: 64px;
+        font-weight: 500;
+        letter-spacing: 4.8px;
+        text-transform: uppercase;
+        color: var(--primary_color);
+        padding: 0 var(--spacing_16) 0 0;
+      }
+      .header-left-gap {
+        display: none;
+      }
+      .header-right-gap {
+        display: block;
+        /*min-width: fit-content;*/
+        flex: auto;
+      }
+      .header-bordered-gap {
+        border-bottom: 1px solid var(--primary_color);
+      }
+      .header-subtitle {
+        display: flex;
+        flex-direction: column;
+        padding: 0 var(--spacing_16) var(--spacing_16) 0;
+      }
+      .header-surname {
+        font-family: Mohave;
+        font-size: 36px;
+        font-weight: 500;
+        letter-spacing: 2.4px;
+        text-transform: uppercase;
+        color: var(--gray_color_900);
+        white-space: nowrap;
+        padding-right: var(--spacing_16);
+      }
+      .header-job-position {
+        font-family: 'Raleway';
+        font-size: 24px;
+        font-weight: 500;
+        color: var(--gray_color_900);
+        border-left: var(--spacing_8) solid var(--primary_color);
+        padding-left: var(--spacing_16);
+      }
+
+      @media (min-width: 1024px) {
+        .header-name {
+          padding-left: var(--spacing_16);
+        }
+        .header-surname {
+          padding-left: var(--spacing_16);
+        }
+        .header-left-gap {
+          display: block;
+          min-width: 100px;
+        }
+        .header-subtitle {
+          flex-direction: row;
+          align-items: center;
+        }
+      }
+
+      /* aside styles */
+      .aside-area {
+        display: flex;
+        flex-direction: column;
+        text-align: left;
+        gap: var(--spacing_64);
+      }
+      .aside-section {
+        display: flex;
+        flex-direction: column;
+        font-family: Roboto, sans-serif;
```

Comment:
Remove  it, we use by default Raleway font

Reply:
OK